OpenSim/Actuators/Millard2012EquilibriumMuscle.cpp

```diff
@@ -156,6 +156,12 @@ void Millard2012EquilibriumMuscle::extendFinalizeFromProperties()
     penMdl.set_optimal_fiber_length(getOptimalFiberLength());
     penMdl.set_pennation_angle_at_optimal(getPennationAngleAtOptimalFiberLength());
     penMdl.set_maximum_pennation_angle(get_maximum_pennation_angle());
+    try {
+        penMdl.finalizeFromProperties();
+    } catch (const SimTK::Exception::Base& ex) {
+        penMdl.setDefaultProperties(); //avoid throwing again
+        OPENSIM_THROW(Exception, ex.getMessage());
```

Comment:
Also you can rethrow the existing exception with just a `throw;` without recreating an exception with the same message.

Reply:
Disregard this comment now.